From: Stanislaw Kardach <stanislaw.kardach@cavium.com>
Date: Thu, 3 Mar 2016 19:04:07 +0100
Subject: [PATCH] Build bootstrap image for arm64

For now the arch is hardcoded for simplicity. We should detect this
but how? If we're multi-arch then one bootstrap image should be built
per supported arch. This requires Fuel refactoring.
---
 .../0001-Patches-for-OPNFV.patch        | 9 +++++++++
 1 file changed, 9 insertions(+)

diff --git a/build/f_repos/patch/fuel-main/0001-Patches-for-OPNFV.patch b/build/f_repos/patch/fuel-main/0001-Patches-for-OPNFV.patch
--- a/build/f_repos/patch/fuel-main/0001-Patches-for-OPNFV.patch
+++ b/build/f_repos/patch/fuel-main/0001-Patches-for-OPNFV.patch
@@ -45,3 +45,12 @@
  # Enable iptables
  systemctl enable iptables.service
+@@ -484,7 +490,7 @@
+         local ret=1
+         echo ${bs_progress_message} >&2
+         set_ui_bootstrap_error "${bs_progress_message}" >&2
+-        if fuel-bootstrap -v --debug build --activate >>"$bs_build_log" 2>&1; then
++        if fuel-bootstrap -v --debug build --target_arch arm64 --activate >>"$bs_build_log" 2>&1; then
+           ret=0
+           fuel notify --topic "done" --send "${bs_done_message}"
+         else
 @@ -546,6 +560,16 @@ systemctl start ntpd
